docker-compose.yml

```diff
@@ -14,6 +14,6 @@ services:
     expose:
       - "25565"
     environment:
-      INFRARED_DEBUG: false
-      INFRARED_COLOR: true
+      INFRARED_DEBUG: "false"
+      INFRARED_COLOR: "true"
```

Comment:
These two are outdated and I forgot to remove them. Would you be so kind and delete them for me?

Reply:
No problem. Fixed in 81a77eb.